SMPyBandits/Policies/GreedyOracle.py

```diff
@@ -0,0 +1,34 @@
+from .IndexPolicy import IndexPolicy
+import numpy as np
+
+class GreedyPolicy(IndexPolicy):
+    """ Greedy Policy for rotting bandits.
+    Selects arm with best last value.
+    Reference: [Heidari , Kearns & Roth, 2016]
+    """
+    def __init__(self, nbArms):
+        super(GreedyPolicy, self).__init__(nbArms)
+        self.arms_history = [np.array([]) for arm in range(nbArms)]
```

Comment:
Probably a bad idea to use `np.array([])` : if the arrays have dynamic sizes, Python lists are more (time) efficient!